### DISCUSSION TITLE
bot will not active while not building

### DISCUSSION THREAD

**Discussion by yuchinchenTW** [discordjs/discord.js#6727] [Q&A] [answered]
So I'm new here,  my discord bot can reply to the slash command while I am building my program but when I stop building the program, the bot will not reply anymore

Anyone know how to fix that?

I do follow the step by step guide
https://discordjs.guide/creating-your-bot/#creating-configuration-files

**Comment by monbrey** [comment, marked as answer]
I'm unsure what you mean by "building" but yes, like all applications, it needs to be running in order to function.

The [#resources](https://canary.discord.com/channels/222078108977594368/729580210634358804/879828812697829448) channel in the discord.js server lists some hosting providers you can use.

**Reply by yuchinchenTW** [reply]
thx

> The [#resources](https://canary.discord.com/channels/222078108977594368/729580210634358804/879828812697829448) channel in the discord.js server lists some hosting providers you can use.

thx but the link do have some issue I cannot enter the channel

**Reply by ImRodry** [reply]
You gotta join https://discord.gg/djs

**Reply by yuchinchenTW** [reply]
I use this as host https://uptimerobot.com/
